masonry/src/tracing_backend.rs

```diff
@@ -142,14 +142,11 @@ pub(crate) fn try_init_test_tracing() -> Result<(), SetGlobalDefaultError> {
 
 /// Initialise tracing for an end-user application.
 pub(crate) fn try_init_tracing() -> Result<(), SetGlobalDefaultError> {
-    // Default level is DEBUG in --dev, INFO in --release, unless a level is passed.
+    // Default level is INFO unless a level is passed.
+    // We tried using DEBUG in --dev, but it was too noisy.
     // DEBUG should print a few logs per low-density event.
     // INFO should only print logs for noteworthy things.
-    let default_level = if cfg!(debug_assertions) {
-        LevelFilter::DEBUG
-    } else {
-        LevelFilter::INFO
-    };
+    let default_level = LevelFilter::INFO;
```

Comment:
This is a slightly weird change to have in this PR. I don't fundamentally disagree with it, but it seems quite orthogonal

Reply:
The main reason is that the log noise was making the printed text harder to see (though less so now that I've added the full debug print).

Reply:
I rolled it back for now.